src/h3/api/basic_int.pyi

```diff
@@ -0,0 +1,84 @@
+from typing import Dict, Iterable, List, Optional, Set, Tuple
+
+from typing_extensions import Literal
+
+from . import _api
+
+H3Cell = int
+H3Edge = int
+UnorderedH3Cell = Set[H3Cell]
+
+# Should be no changes in each API file below this line
+
+AreaUnits = Literal['km^2', 'm^2', 'rads^2']
+LengthUnits = Literal['km', 'm', 'rads']
+Resolution = int
+GeoBoundary = Tuple[Tuple[float, float], ...]
+KRange = int
+
+def versions() -> Dict[str, str]: ...
+def string_to_h3(h: str) -> int: ...
+def h3_to_string(x: int) -> str: ...
+def num_hexagons(resolution: Resolution) -> int: ...
+def hex_area(resolution: Resolution, unit: AreaUnits = 'km^2') -> float: ...
```

Comment:
Is it possible to avoid repeating the entire API definition after defining H3Cell each time? (As below in the PR?)